src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -1059,6 +1069,34 @@ public static FormulaValue OptionSetValueToLogicalName(IRContext irContext, Opti
             var optionSet = args[0];
             var logicalName = optionSet.Option;
             return new StringValue(irContext, logicalName);
+        }
+
+        public static FormulaValue PlainText(IRContext irContext, StringValue[] args)
+        {        
+            //Decode html specific characters
+            string text = WebUtility.HtmlDecode(args[0].Value.Trim());
+
+            // Replace header/script/style tags with empty text.
+            text = _headerTagRegex.Replace(text, string.Empty);
+            text = _scriptTagRegex.Replace(text, string.Empty);
+            text = _styleTagRegex.Replace(text, string.Empty);
+
+            // Remove all comments.
+            text = _commentTagRegex.Replace(text, string.Empty);
+
+            // Insert empty string in place of <td>
+            text = _tdTagRegex.Replace(text, string.Empty);
+
+            //Replace <br> or <li> with line break
+            text = _lineBreakTagRegex.Replace(text, Environment.NewLine);

```

Comment:
>text = _lineBreakTagRegex.Replace(text, Environment.NewLine); [](http://example.com/codeflow?start=12&length=61)

We should have tests for this case. And for the next one.

Reply:
added tests.